src/components/Projects/chatGPT-Projects.html

```diff
@@ -83,4 +83,3 @@ <h3>Project 3</h3>
   crossorigin="anonymous"></script>
   
 </body>
```

Comment:
```suggestion
</body>
</html>
```
This was removed but the original file had the start of the HTML code. On the next go-round this file will be removed anyway.

Reply:
Yeah eslint was complaining here because technically you dont need this closing tag as the first one is self closing.

Reply:
*smacks forehead* yup - thanks for the reminder